fastlane/spec/unused_options_spec.rb

```diff
@@ -4,10 +4,12 @@
       let (:all_exceptions) { %w(pilot appstore cert deliver gym match pem produce scan sigh snapshot supply testflight mailgun testfairy ipa import_from_git hockey deploygate crashlytics artifactory appledoc slather screengrab download_dsyms notification frameit set_changelog register_device register_devices latest_testflight_build_number app_store_build_number sh swiftlint) } # rubocop:disable Metrics/LineLength
 
       Fastlane::ActionsList.all_actions do |action, name|
-        next unless action.available_options.kind_of?(Array)
-        next unless action.available_options.last.kind_of?(FastlaneCore::ConfigItem)
-
         it "No unused parameters in '#{name}'" do
+          allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return("./fastlane")
+
+          next unless action.available_options.kind_of?(Array)
+          next unless action.available_options.last.kind_of?(FastlaneCore::ConfigItem)
```

Comment:
The call to `available_options` will use variable called `containing`, which is defined as `containing = Helper.fastlane_enabled? ? FastlaneCore::FastlaneFolder.path : '.'` after the change.

So in this case the `FastlaneCore::FastlaneFolder.path` has to be mocked with `allow` before call to `available_options`.

I tried to move `allow` to outer scope. I have put it in a `before do/end` block right inside `describe Fastlane`, `describe Fastlane::Action` as well as `describe "No unused options"`. 

For all those cases the `before do/end` block wasn't called 😕  and `available_options` would get `nil` when resolving `containing` variable...

The block would only be called when put inside `it` spec, but in that case I had to move the `available_options` call inside the `it` block too.

I'm sure I'm missing something here, can't figure out why the `before do/end` block would not get called when put inside `describe` block.